docs/building-examples.rst

```diff
@@ -0,0 +1,82 @@
+Building example designs
+========================
+
+Before building any example, set the installation directory to match what you
+set it to earlier, for example:
+
+.. code-block:: bash
+   :name: export-install-dir
+
+   export INSTALL_DIR=~/opt/symbiflow
+
+Select your fpga family:
+
+.. tabs::
+
+   .. group-tab:: Artix-7
```

Comment:
Could we maybe extend tuttest to understand `.. group-tab::` in some way?

Reply:
Currently, we are supporting this directive. It is treated as a container for other directives. 
I believe that this is the correct approach, since `group-tab` can contain more directives. 

See: 
- https://docs.zephyrproject.org/latest/getting_started/index.html#install-dependencies
- https://github.com/zephyrproject-rtos/zephyr/blob/master/doc/getting_started/index.rst#install-dependencies